### ISSUE TITLE
color used for flood fill is not added to color history 

### ISSUE THREAD

**Issue by jeisner** [mypaint/mypaint#473]
Usually when you use a new color to draw, it is added to the "recent colors" list that is pulled up by the Color History Popup command (X).

**However**, this is not true when you use a new color for flood fill.   (To replicate: Press U to select that tool, press C and select a new color, click to fill with that color.)  

I assume this is a bug.  (Of course it is often easy to use the eyedropper to match the color of an existing flood fill, but not if you've hit undo ... as happens when you try one fill color, undo it, try another, and then decide you wanted to go back to the first one.)

**Comment by achadwick** [comment]
Patches welcome. This would be quite easy to achieve: the coordinating code for flood fill lives in `gui/fill.py`, and an example of out-of-band setting of colour history can be found in the code for handling colour drops in `gui/drawwindow.py`.

**Comment by cortexer** [comment]
I cannot reproduce this problem. Chosen colours appear in the colour history popup after being used by the flood fill tool.

**Comment by achadwick** [comment]
@cortexer Confirmed. It's not happening with 1.2.0-beta.2.3 from git at f40bd31.

@jeisner What version are you testing with? Can you give some stepwise instructions telling me how to reproduce this issue please?

**Comment by jeisner** [comment]
Thanks @cortexer for looking at it.
I am using 1.2.0-beta.2+gitexport.7169334, from @chadwick's PPA.

Oddly, I'm also no longer able to reproduce it by following the instructions that I posted.
Since none of us can reproduce it, probably best to close the issue for now.
Although I am sure that I tested those instructions at least twice at the time.

Could it be an intermittent issue?  When I posted, I had been playing around with flood fill for a while, so maybe I'd gotten my session into a funny state?

Or conceivably I was using an earlier version when I posted, and it has since been fixed without notice?  (Seems unlikely.)
